src/objective-c/tests/run_one_test_bazel.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
where is this file used?

Reply:
In the future... where Xcode is upgraded and `tvos_unit_test` actually works

Reply:
ok... maybe add a TODO on the top that explains it